index.html

```diff
@@ -9,51 +9,229 @@
+                    </a>
+                </li>
+                <li>
+                    <a href="#" class="social twiter">
+                        <img src="./images/mail.png" alt="Email"/>
+                    </a>
+                </li>
+                <li>
+                    <a href="#" class="social github">
+                        <img src="./images/Vector.png" alt="GitHub"/>
+                    </a>
+                </li>
+                <li>
+                    <a href="#" class="social peace">
+                        <img src="./images/fingers.png" alt="Peace"/>
+                    </a>
+                </li>
+            </ul>
+        </div>
+   </div>
+   
+    <!--Work Section-->
+    <section class="work-section">
+        <section class="tonic-frontpage padding">
+            <div class="frontpage-icon">
+                <img src="./images/Snapshoot-Portfolio.png" alt="Portfolio">
+            </div>
+            <div class="tonic">
+                <h2>Tonic</h2>
+                <p class="tonic-section-header">
+                    <span class="canopy">CANOPY</span> <img src="./images/Counter.png" alt="counter" class="counter"/>
+                    <span class="back-end-dev">Back End Dev</span> <img src="./images/Counter.png" alt="counter"/>
+                    <span class="year">2022</span>
+                </p>
+                
+                <p class="tonic-details">A daily selection of privately personalized reads: no accounts or sign-ups required</p>
+            </div>
+    
+            <ul class="programing-languages">
+                <li><a href="#">html</a></li>
+                <li><a href="#">css</a></li>
+                <li><a href="#">javaScript</a></li>
+            </ul>
+    
+            <button type="button">See Project</button>
+        </section>
+
+        <!--Multi-post Availability Section-->
+        <section class="multi-post-availability padding">
+            <div class="availability-icon">
+                <img src="./images/Availability.png" alt="Portfolio">
+            </div>
+            <div class="tonic">
+                <h2>Multi-Post <br>Stories</h2>
+                <p class="tonic-section-header">
+                    <span class="canopy">CANOPY</span><img src="./images/Counter.png" alt="counter" class="counter"/>
+                    <span class="back-end-dev">Back End Dev</span><img src="./images/Counter.png" alt="counter"/>
+                    <span class="year">2022</span>
+                </p>
+                
+                <p class="tonic-details">A daily selection of privately personalized reads: no accounts or sign-ups required</p>
+            </div>
+    
+            <ul class="programing-languages">
+                <li><a href="#">html</a></li>
+                <li><a href="#">css</a></li>
+                <li><a href="#">javaScript</a></li>
+            </ul>
+            <button type="button">See Project</button>
+        </section>
```

Comment:
All your cards don't match the Figma template you are following. Please make changes to all the cards.
| Figma template  | Your design |
| ------------- | ------------- |
| ![card](https://user-images.githubusercontent.com/33268927/170697941-846dc4a2-55b9-4135-b6f8-27f7d1f8bf4a.png) | ![issue](https://user-images.githubusercontent.com/33268927/170697984-f8feb6e3-0118-48d9-8ca5-75b7fde2affb.png) |

Reply:
@muneebulrehman I have worked on the reviewed feedback. I appreciate the feedback because it made me go the extra mile. However, I think the very first picture was matched accurately if only the pictures were considered.